### ISSUE TITLE
msktutil fails when default_ccache_name is KEYRING

### ISSUE THREAD

**Issue by mikes-gh** [msktutil/msktutil#138]
Not sure this ever got posted here

RHEL issue explains it well enought

https://bugzilla.redhat.com/show_bug.cgi?id=1471144

It affects a default install of CentOS7+ and RHEL7+

My notes

```
# msktutil fails to auth due to following bug
# https://bugzilla.redhat.com/show_bug.cgi?id=1471144
# Comment out default_ccache_name
sed -i -e 's/default_ccache_name = KEYRING:persistent:%{uid}/#default_ccache_name = KEYRING:persistent:%{uid}/g' /etc/krb5.conf

```

**Comment by mikes-gh** [comment]
Ok I just saw

https://github.com/msktutil/msktutil/commit/b169f2c4f456a31476e4a31d476a670acdfe1b4f

Even though its 1.1 it was from 2014 and I only just got a new update from epel so it should be fixed will report back

**Comment by mproehl** [comment]
This should have been fixed in 2014 with commit fc6438bc7622385dfd60e14e2d090d0a91423cb4

**Comment by mikes-gh** [comment]
Just saw that sorry posted wrong link.
I think the issue was epel only just got updated with that code?
Is that right?

**Comment by mikes-gh** [comment]
It works.  :smile: Thanks  removing workaround from my notes now new package is available

**Comment by mproehl** [comment]
I think the bugfix has been released in version 1.0. According to EPEL's changelog their package has been updated to 1.0 on Jan 08 2017. But https://bugzilla.redhat.com/show_bug.cgi?id=1471144 refers to 0.5.1-2.el7 which still included the bug.

**Comment by mikes-gh** [comment]
I was using 
Extra Packages for Enterprise Linux 7 - x86_64
And it only just updated to 1.1 from 0.5.1 (I do regular yum update)
Strange maybe their 1.0 package had an issue.

Anyway glad its sorted thankyou for a great tool
